### DISCUSSION TITLE
Is it possible to skip PKGBUILD check by default as it is in `yay`?

### DISCUSSION THREAD

**Discussion by SinTan1729** [Morganamilo/paru#498] [Q&A]
I know that it's recommended to always check the PKGBUILD but most of the time I'm installing sofrware I already trust, and would like the default option for PKGBUILD check to be N and only show it when I specifically ask, like it is in `yay`. Is it possible to do that?

**Comment by Morganamilo** [comment]
See review in the man page.

**Comment by jasursadikov** [comment]
I am also curious how it is being done :)
